frontend/src/actionCreators/accessRequestActionCreator.tsx

```diff
@@ -114,16 +122,50 @@ export const getAccessRequestsAction = (params: API.IPaginateAccessRequests) =>
     .get(ENVIRONMENT.apiUrl + API.REQUEST_ACCESS_LIST(params), createRequestHeader())
     .then((response: AxiosResponse) => {
       dispatch(success(actionTypes.GET_REQUEST_ACCESS, response.status));
-      const clearAction: IStoreAccessRequestsAction = {
+      dispatch({
         type: actionTypes.STORE_ACCESS_REQUESTS,
-        pagedAccessRequests: { page: 0, quantity: 0, total: 0, items: [] },
-      };
-      dispatch(clearAction); //TODO: this should not be necessary.
-      dispatch(storeAccessRequests(response.data));
+        pagedAccessRequests: { page: 0, quantity: 0, total: 0, items: response.data },
+      });
+
       dispatch(hideLoading());
     })
     .catch((axiosError: AxiosError) =>
       dispatch(error(actionTypes.GET_REQUEST_ACCESS, axiosError?.response?.status, axiosError)),
     )
     .finally(() => dispatch(hideLoading()));
 };
+
+export const getAccessRequestsFilterAction = (filters: {
+  [key: number]: IColumnFilter;
+}): IFilterAccessRequestsAction => ({
+  type: actionTypes.FILTER_REQUEST_ACCESS_ADMIN,
+  filters,
+});
+
+export const getAccessRequestsSelectAction = (
+  selections: string[] = [],
+): ISelectAccessRequestsAction => ({
+  type: actionTypes.SELECT_REQUEST_ACCESS_ADMIN,
+  selections,
+});
+
+export const getAccessRequestsSortAction = (sorting: ISort[] = []): ISortAccessRequestsAction => ({
+  type: actionTypes.SORT_REQUEST_ACCESS_ADMIN,
+  sorting,
+});
+
+export const getAccessRequestsDeleteAction = (id: number) => (dispatch: Dispatch<any>) => {
```

Comment:
same comment here about dispatching the action to set isFetching.

This will need updating anyways when you make the api change to start passing an access request during deletion.